src/workerd/api/streams/encoding.c++

```diff
@@ -48,9 +49,9 @@ TextDecoderStream::TextDecoderStream(jsg::Ref<TextDecoder> decoder,
 jsg::Ref<TextDecoderStream> TextDecoderStream::constructor(
     jsg::Lock& js, jsg::Optional<kj::String> label, jsg::Optional<TextDecoderStreamInit> options) {
 
-  auto decoder = TextDecoder::constructor(js, kj::mv(label), options.map([](auto& opts) {
+  auto decoder = TextDecoder::constructor(js, kj::mv(label), options.map([&js](auto& opts) {
     return TextDecoder::ConstructorOptions{
-      .fatal = opts.fatal.orDefault(true),
+      .fatal = opts.fatal.orDefault(!FeatureFlags::get(js).getPedanticWpt()),
```

Comment:
Nit: simply because the context here is not clear and the the `!` in that is easy to miss.

```suggestion
      // Previously this would default to true. The spec requires a default
      // of false, however. When the pedanticWpt flag is not set, we continue
      // to default as true.
      .fatal = opts.fatal.orDefault(!FeatureFlags::get(js).getPedanticWpt()),
```